pkg/solana/chainreader/chain_reader.go

```diff
@@ -154,13 +153,21 @@ func (s *ContractReaderService) GetLatestValue(ctx context.Context, readIdentifi
 
 	values, ok := s.lookup.getContractForReadIdentifiers(readIdentifier)
```

Comment:
I guess this is a pre-existing issue, but this method name seems very misleading. The name sounds like the input would be a slice of read identifiers and the output would be a contract. Shouldn't it be called `getReadValuesForReadIdentifier`?